Stripe/StripeiOS/Source/STPAddCardViewController.swift

```diff
@@ -687,7 +697,11 @@ public class STPAddCardViewController: STPCoreTableViewController, STPAddressVie
         case STPPaymentCardSection.stpPaymentCardNumberSection.rawValue:
             cell = paymentCell
         case STPPaymentCardSection.stpPaymentCardScannerSection.rawValue:
-            cell = scannerCell
+            if #available(macCatalyst 14.0, *) {
+                cell = scannerCell
+            } else {
+                return UITableViewCell()
```

Comment:
What's the behavior in this case?

Reply:
Should never happen — the "Scan card" button won't appear, so `isScanning` will never be set, so `numberOfRowsInSection` for the scanner section will be 0.